### ISSUE TITLE
cannot serialize a bytes object larger than 4 GiB in Popen

### ISSUE THREAD

**Issue by lhwcv** [keras-team/autokeras#345]
### Bug Description

When  the data loader hold large dataset(>4G) in memory, it  will happen.
Maybe we can do in current process instead of subprocess?  [here in search.py](https://github.com/jhfjhfj1/autokeras/blob/master/autokeras/search.py#L190-L193)

### Reproducing Steps
    You can use the similiar code to reproduce,  where the loader hold >4G  data in memmory

     cnn = CnnModule(classification_loss,   Accuracy, {}, config_dict['model_outdir'], True)  
     train_loader = get_train_loader()  
     test_loader = get_val_loader()  
     cnn.fit(17,(128,32,32,18), train_loader, test_loader, 12*60*60)

**Comment by tl-yang** [comment]
Hi @lhwcv . Thank you so much for the report. We're trying to figure out a good solution for this issue. If you have any thought please let us know!
